src/main/java/net/ltxprogrammer/changed/process/ProcessTransfur.java

```diff
@@ -688,7 +690,13 @@ public static void transfur(LivingEntity entity, Level level, TransfurVariant<?>
         if (entity == null)
             return;
         if (entity.isDeadOrDying())
-            return; // To prevent most bugs, entity has to be alive to transfur
+            return;//
+        // To prevent most bugs, entity has to be alive to transfur
+        if(entity instanceof Player player){
+            PatienceCompatibility compatibility = new PatienceCompatibility((ServerPlayer) player);
```

Comment:
Unchecked cast to `ServerPlayer` is not guaranteed at this point in execution. And see my other comment on having to initialize "PatienceCompatibility" every time you need to check.